data-r2dbc/src/main/java/io/micronaut/data/r2dbc/connection/DefaultR2dbcReactorConnectionOperations.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.data.r2dbc.connection;
+
+import io.micronaut.context.annotation.EachBean;
+import io.micronaut.context.annotation.Parameter;
+import io.micronaut.core.annotation.Internal;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.data.connection.ConnectionDefinition;
+import io.micronaut.data.connection.ConnectionStatus;
+import io.micronaut.data.connection.support.AbstractReactorConnectionOperations;
+import io.micronaut.data.r2dbc.config.DataR2dbcConfiguration;
+import io.micronaut.data.r2dbc.operations.R2dbcSchemaHandler;
+import io.micronaut.data.runtime.multitenancy.SchemaTenantResolver;
+import io.r2dbc.spi.Connection;
+import io.r2dbc.spi.ConnectionFactory;
+import org.reactivestreams.Publisher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.function.Function;
+
+/**
+ * The reactive R2DBC connection operations implementation.
+ *
+ * @author Denis Stepanov
+ * @since 3.3
```

Comment:
Not important probably, but should this be since 4.0 and 2023 in license header? Or this was just renamed existing class?

Reply:
I just renamed the file and git didn't catch it somehow `R2Dbc` -> `R2dbc`